java-agent/src/main/java/io/opentelemetry/auto/bootstrap/AgentBootstrap.java

```diff
@@ -71,24 +73,26 @@ public static void agentmain(final String agentArgs, final Instrumentation inst)
       startMethod.invoke(null, inst, bootstrapURL);
     } catch (final Throwable ex) {
       // Don't rethrow.  We don't have a log manager here, so just print.
+      System.err.println("ERROR " + thisClass.getName());
       ex.printStackTrace();
     }
   }
 
   private static synchronized URL installBootstrapJar(final Instrumentation inst)
       throws IOException, URISyntaxException {
-    URL bootstrapURL = null;
+    URL ddJavaAgentJarURL = null;
```

Comment:
Old name was better :)

Reply:
Even worse thing is that I created 3rd name for the same var. It also called `bootstrapJarLocation` :
https://github.com/DataDog/dd-trace-java/blob/03ec987180f3f79bb355690ac080c344e9bee906/dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/DatadogClassLoader.java#L31 which I think is better.

I think it should be one name for this var across all methods. 
Please help me which name to choose:
- bootstrapURL
- bootstrapJarLocation
- ddJavaAgentJarURL
- something else ?

Reply:
i like `ddJavaAgentJarURL`, i think it is very descriptive. i did fix it just now to remove the `dd` prefix in this repo. @iNikem was that your concern?

Reply:
Yes, `dd` was my problem :)